### PR TITLE
chore(refs T35059): Optimize Translations (keys and String)

### PR DESCRIPTION
**Pull request by salisdemos**
Find a better Wording for the errormessage, where a name has to be unique.

And even though the transkey is in use only in the customer area, The key itself should be generic enough to be used in other contexts.

**Ticket:** https://yaits.demos-deutschland.de/T35059

### CODE REVIEW

File: translations/messages+intl-icu.de.yml [hunk 1]
```diff
@@ -680,10 +680,24 @@ consultation.notification.mail.on.token.creation.confirmation: "Ein Zugangscode
...
 consultation.notification.mail.subject.on.token.creation: "Ihre Stellungnahme zum Verfahren {procedureName} ist eingegangen"
 
+contact: "Kontakt"
 contact.data: "Kontaktdaten"
+contact.add: "Kontakt hinzufügen"
+contact.change: "Kontakt bearbeiten"
+contact.delete: "Kontakt löschen"
+contact.deleted: "Der Kontakt wurde gelöscht."
+contact.new: "Neuen Kontakt anlegen"
+contact.no: "Keine Kontakte angelegt"
```
Comment (muellerdemos) [inline translations/messages+intl-icu.de.yml:690]: Could we tweak this translation key a little bit. Imho I think the key `contact.no` is not really transporting the translation message. Maybe something like `contact.no_entries` or `contact.no_new_entries`?

**Review by muellerdemos — CHANGES_REQUESTED** [review]
just a tiny remark

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (salisdemos) [reply translations/messages+intl-icu.de.yml]: good catch b9dbaffc0a16edd4bd35e58abdaac2bac8a3f44b

**Review by muellerdemos — APPROVED** [review]
thanks - lgtm

**Pull request merged**